README.md

```diff
@@ -1,6 +1,15 @@
 # OpenMATERIAL
-3D model exchange format with physical material properties for virtual development, test and validation of automated driving.
+A new ASAM standard with the target to develop a 3D model exchange format with physical material properties for virtual development, test and validation of automated driving.
```

Comment:
A new ASAM standard with the target of developing a 3D model exchange format with physical material properties for virtual development, test, and validation of automated driving.